proto/internalapi/scanner/v4/indexer_service.proto

```diff
@@ -44,6 +45,14 @@ message GetOrCreateIndexReportRequest {
   }
 }
 
+message StoreIndexReportRequest {
+  string hash_id = 1;
+  string cluster_name = 2;
+  Contents contents = 3;
+}
+
+message StoreIndexReportResponse { }
```

Comment:
If there isn't a grpc status code to indicate that a record was updated, skipped, etc. could be beneficial to add some sort of status / message to the response so that we can log the outcome in Central.